core/storage/user/gae_models.py

```diff
@@ -2141,10 +2143,15 @@ class PendingDeletionRequestModel(base_models.BaseModel):
     # IDs of all the private collections created by this user.
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
 
-    # A dict mapping story IDs to pseudonymous user IDs. For each story, we
-    # use a different pseudonymous user ID. Note that all these pseudonymous
+    # A dict mapping model IDs to pseudonymous user IDs. Each typeof activity
```

Comment:
It's not possible that these models exist in production yet, right?

Also: typeof --> type of, isgrouped --> is grouped.